salt/edx/tests/test_edxapp_running.sls

```diff
@@ -0,0 +1,86 @@
+#!jinja|yaml
+
+{# Test to verify that the edxapp is funcitional and configured with some of
+our configurations. Test the following:
+- edxapp supervisor services are running
+- edxapp instance can reach select services
+- services that edxapp relies on locally
+- edxapp:cms service is only running on edx-draft
+- AWS EBS data repo is mounted
+- spot check custom vars are in lms.{env|auth} #}
+
+
+{% set supervisor_services = [
+    'edxapp:lms',
+    'forum',
+    'xqueue',
+    'xqueue_consumer'
+  ] %}
+
+{% set socket_connections = {
+    'ssl': 'tcp://0.0.0.0:443',
+    'lms': 'tcp://0.0.0.0:8000',
+    'gitreload_service': 'tcp://0.0.0.0:8095',
+    'xqueue': 'tcp://0.0.0.0:18040',
+    'forum': 'unix:///edx/var/forum/forum.sock',
```

Comment:
This should use the TCP port that the forum is listening on since we aren't using the unix socket. You may need to refer to edx/configuration for the appropriate port number.